packages/access-triangle/package.py

```diff
@@ -0,0 +1,86 @@
+# Copyright 2013-2024 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+# ----------------------------------------------------------------------------
+#
+#     spack install access-triangle-git
+#
+# You can edit this file again by typing:
+#
+#     spack edit access-triangle-git
+#
+# ----------------------------------------------------------------------------
+
+from spack.package import *
+import glob
+import os
+
+
+class AccessTriangle(MakefilePackage):
+    """Example Spack package for the ISSM Triangle library."""
```

Comment:
It's not an example. You may want to mention who created the triangle library originally.